pyproject.toml

```diff
@@ -36,7 +36,7 @@ include=[
 [tool.poetry.dependencies]
 python = ">=3.8,<4.0"
 redis = ">=3.5.3,<5.0.0"
-pydantic = ">=1.10.2,<2.1.0"
+pydantic = ">=1.10.2,<3.0.0"
```

Comment:
I'd remove the upper bound entirely, see this article: https://arc.net/l/quote/txkfghki

Reply:
I don't agree with that.
Pydantic v3 would have breaking changes, see: [Pydantic V3 and beyond](https://docs.pydantic.dev/latest/version-policy/#pydantic-v3-and-beyond)
Nothing promises it will be compatible with older versions. Especially this library is still both Pydantic v1 and v2.

Reply:
yes, that'd be the cause, but there might be a chance where this library wouldn't be affected by the breaking changes :)
And if you're affected by the breaking changes you can pin your side 😊